generator/sequence_transform_data.py

```diff
@@ -386,14 +374,17 @@ def serialize_outputs(
     completions_offset = 0
     max_completion_len = 0
 
-    for output in sorted(outputs, key=len, reverse=True):
+    output_list = list(outputs)
+    output_list.sort()
+
+    for output in sorted(output_list, key=len, reverse=True):
```

Comment:
I stabilized the sorting of the completions so that it would be easier to detect problems when I changed things.